package.json

```diff
@@ -36,61 +36,61 @@
     "format:diff": "prettier --config .prettierrc --list-different \"**/*.{js,jsx,ts,tsx,json,astro}\""
   },
   "dependencies": {
-    "@astrojs/check": "^0.9.4",
-    "@astrojs/markdown-remark": "^5.3.0",
-    "@astrojs/mdx": "^3.1.8",
-    "@astrojs/react": "^3.6.2",
-    "@astrojs/tailwind": "^5.1.2",
-    "@asyncapi/react-component": "^2.2.2",
-    "@headlessui/react": "^2.0.3",
-    "@heroicons/react": "^2.1.3",
-    "@parcel/watcher": "^2.4.1",
-    "@stoplight/json-schema-viewer": "^4.7.0",
-    "@stoplight/mosaic": "^1.53.2",
-    "@tailwindcss/typography": "^0.5.13",
-    "@tanstack/react-table": "^8.17.3",
-    "@types/dagre": "^0.7.52",
-    "@types/diff": "^5.2.2",
+    "@astrojs/check": "0.9.4",
```

Comment:
what we gain by having exact version?

Reply:
Some patches could introduce bugs, like the described in #644.

So, we have more control using exact version.